packages/server/graphql/queries/tasks.ts

```diff
@@ -19,17 +20,28 @@ export default {
     teamId: {
       type: GraphQLID,
       description: 'The unique team ID'
+    },
+    archived: {
+      type: GraphQLBoolean,
+      description: 'true if only archived tasks are returned; false otherwise',
+      defaultValue: false
     }
   },
-  async resolve(_source, {teamId}, {authToken, dataLoader}: GQLContext) {
+  async resolve(_source, {first, after, teamId, archived}, {authToken, dataLoader}: GQLContext) {
     // AUTH
     const viewerId = getUserId(authToken)
     if (teamId && !isTeamMember(authToken, teamId)) {
       standardError(new Error('Team not found'), {userId: viewerId})
       return connectionFromTasks([])
     }
     const teamIds = teamId ? [teamId] : authToken.tms || []
-    const tasks = await dataLoader.get('userTasks').load({userId: viewerId, teamIds})
+    const tasks = await dataLoader.get('userTasks').load({
+      first: first,
+      after: after,
+      userId: viewerId,
```

Comment:
before in the team archive view, it showed the archived tasks for all users on a particular team.
with this PR, it only shows tasks for that team for the current viewer.
i think you've probably fixed this in the user filter branch?
just wanna make sure that functionality is still there before merging

Reply:
Nice catch! Yeah you are right it's fixed in user filter branch. Let me cherry-pick those changes here.